### ISSUE TITLE
User component form does not meet accessibility standards

### ISSUE THREAD

**Issue by collincahill** [Jropp/jason-ropp-js-iobp#44]
## Main Goal
Make the Edit/Create user form accessible according to the [MDN Guide for form structure](https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/How_to_structure_an_HTML_form)

## Notes
Mostly looking for label tags and input ids.
